index.js

```diff
@@ -8,13 +8,17 @@ const termImg = require('term-img');
 const PIXEL = '\u2584';
 const readFile = util.promisify(fs.readFile);
 
-async function render(buffer) {
+function asPercent(value) {
+	return `${Math.round(value * 100)}%`;
+}
+
+async function render(buffer, factor) {
```

Comment:
Instead of directly define a `factor` argument, it's better to define an `options` one being `factor` an attribute, this way it would be possible to add `width` and `height` options without changing the function signature.

Reply:
I originally had it this way, but then I thought: YAGNI, so let's keep things simple for now 😉

Reply:
PS: Feel free to improve this PR.

Reply:
> YAGNI

Lol, didn't know that :-P Ok, I'll try to do some improvements :-)